jscomp/others/js_cast.ml

```diff
@@ -26,4 +26,6 @@
 
 external intOfBool : bool -> int = "%identity"  
 
-external floatOfInt : int -> float = "%identity"  
+external floatOfInt : int -> float = "%identity"
+
+external dictOfObj : 'a Js.t -> 'b Js.Dict.t = "%identity"
```

Comment:
this API is too polymorphic which maybe wrong in some cases, can we move it into your config module